eve/main.yml

```diff
@@ -166,6 +166,29 @@ stages:
         env:
           <<: *global_env
     - ShellCommandWithSecrets: *test_zenko
+    - ShellCommandWithSecrets:
```

Comment:
When rebasing your branch, note that `ShellCommandWithSecrets` will be deprecated in future eve releases. Just use `ShellCommand` instead.